foundation-extensions/places/places-usage-reference.md

```diff
@@ -11,6 +11,7 @@ This document contains usage information for the public functions, classes, and
 * [getNearbyPointsOfInterest](places-usage-reference.md#getNearbyPointsOfInterest)
 * [processRegionEvent](places-usage-reference.md#processRegionEvent)
 * [registerExtension](places-usage-reference.md#registerExtension)
+* [setAccuracyAuthorization](places-usage-reference.md#setAccuracyAuthorization)
 * [setAuthorizationStatus](places-usage-reference.md#setAuthorizationStatus)
```

Comment:
This section may not be required, the left panel should serve the search/navigation purpose

Reply:
Yup! We don't need to have the in-document table of contents since we have the left navigation to do that heavy lifting for us.